### PR TITLE
[docs] spelling and general improvements (part 3)

### PR DESCRIPTION
**Pull request by daniel-lang**
Part 3 of I don't know how many...

The content of [entity-inheritance.md](https://github.com/typeorm/typeorm/tree/master/docs/entity-inheritance.md) is also in [entities.md#entity-inheritance](https://github.com/typeorm/typeorm/blob/master/docs/entities.md#entity-inheritance). Do you want to have two files or one? I would remove one of the two.

Related to #977

### CODE REVIEW

File: docs/decorator-reference.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class User {
 
 `@Column` accept several options you can use:
 
-* `type: ColumnType` - Column type. One of the type listed [above](#column-types).
+* `type: ColumnType` - Column type. One of the [supported column types](./entities.md#column-types).
```
Comment (pleerock) [inline docs/decorator-reference.md]: i guess website will not create a proper non-github link...

File: docs/embedded-entities.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ export class User {
     id: string;
     
     @Column()
-    nameFirst: string;
```
Comment (pleerock) [inline docs/embedded-entities.md]: I called them inverse for some reason, I dont remember why... something with final database column names

File: docs/decorator-reference.md (hunk 1 above)
Comment (daniel-lang) [reply docs/decorator-reference.md]: What do you mean with "non-github link"?

File: docs/embedded-entities.md (hunk 2 above)
Comment (daniel-lang) [reply docs/embedded-entities.md]: Do you want me to revert these changes?

**Comment by daniel-lang** [comment]
Could you tell me, what you think of this
> The content of entity-inheritance.md is also in entities.md#entity-inheritance. Do you want to have two files or one? I would remove one of the two.

**Pull request merged**